cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,26 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+
+    cy.visit('/login');
+
+    cy.url()
+    .should('include', '/login');
+
+    cy.get('h1')
+    .contains('Sign In');
+  });
   it('should provide an ability to log in', () => {
+
+    cy.get('input[type="email"]').type('andreykuzmich91@qa1.com');
+
+    cy.get('input[type="password"]').type('12345678');
+
+    cy.get('button[type = "submit"]').click();
+
+    cy.contains('a', 'aaaaaaaaaaaaaaaaaaaaaaaaaa1').click();
```

Comment:
The `cy.contains` assertion should be more specific. You can use a more accurate selector and assert that the correct username appears.

Reply:
but we have only one link with username on the page=)) i will fix that=)